### PR TITLE
ADEN-9327 Add ad-products types

### PR DESCRIPTION
**Pull request by Bielik20**

### CODE REVIEW

File: src/ad-products/templates/uap/themes/theme.ts [hunk 1]
```diff
@@ -1,38 +1,18 @@
-import { AdSlot, context, PorvataPlayer, utils } from '@ad-engine/core';
-import { BigFancyAdAboveConfig } from '../big-fancy-ad-above';
+import { AdSlot } from '@ad-engine/core';
+import { PorvataPlayer } from '../../..';
```
Comment (nandy-andy) [inline src/ad-products/templates/uap/themes/theme.ts]: Can we make it more explicit where the class is defined? Or it's generating some unnecessary cost?

File: src/ad-products/video/player/porvata/index.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// export * from './porvata';
```
Comment (nandy-andy) [inline src/ad-products/video/player/porvata/index.ts]: Why can't we just remove this file if it's all commented?

Comment (Bielik20) [reply src/ad-products/video/player/porvata/index.ts]: I forgot about it, good catch, fixes 👍

File: src/ad-products/templates/uap/themes/theme.ts (hunk 1 above)
Comment (Bielik20) [reply src/ad-products/templates/uap/themes/theme.ts]: It should have been full path, must have slipped pass me. I have now different webstorm config - it should suggest correct paths by default, but still need some testing. Good catch, fixed 👍

**Pull request merged**